rust/main/chains/hyperlane-sealevel/src/mailbox.rs

```diff
@@ -798,13 +809,24 @@ impl SealevelMailboxIndexer {
             .into_inner();
         let message_id = delivered_message_account.message_id;
 
-        let log_meta = self
-            .delivered_message_log_meta(
-                U256::from(nonce),
-                &valid_message_storage_pda_pubkey,
-                &delivered_message_account.slot,
-            )
-            .await?;
+        // let log_meta = self
+        //     .delivered_message_log_meta(
+        //         U256::from(nonce),
+        //         &valid_message_storage_pda_pubkey,
+        //         &delivered_message_account.slot,
+        //     )
+        //     .await?;
+
+        let log_meta = LogMeta {
+            address: self.program_id.to_bytes().into(),
+            block_number: delivered_message_account.slot,
+            // TODO: get these when building out scraper support.
+            // It's inconvenient to get these :|
+            block_hash: H256::zero(),
+            transaction_id: H512::zero(),
+            transaction_index: 0,
+            log_index: U256::zero(),
```

Comment:
iiuc this is unused now but if it's not, would need to update too

Reply:
It was defaulted to zero before, so, should be good.